toPDFviaQuarto.qmd

```diff
@@ -0,0 +1,169 @@
+---
+title: Poseidon package specification \vspace{-1cm}
```

Comment:
I would suggest to put the version right into the title. See for example https://samtools.github.io/hts-specs/VCFv4.2.pdf.

Reply:
Nice! I took some inspiration from that title :+1: